apps/site/src/app/(main)/apply/sections/Form/Form.tsx

```diff
@@ -25,6 +26,12 @@ export default function Form() {
 	): Promise<void> => {
 		// Disable native post submission
 		event.preventDefault();
+
+		if (hasDeadlinePassed()) {
+			window.location.reload();
```

Comment:
Chore: Could you include a comment to indicate this is to show the messaged that applications have closed.